rfc/rfc006-distributed-registry.md

```diff
@@ -379,7 +380,79 @@ The reference MUST resolve to a contact information service as described above.
 Since the information is self-proclaimed and not authenticated or verified in any way, applications MUST treat it as
 untrusted, with great care. Failing to do so could make the operator of the node vulnerable for spoofing and other attacks. 
 
-## 5. Supported Cryptographic Algorithms and Key Types
+## 5. Conflict resolution
+
+§3.4 of [RFC004](rfc004-verifiable-transactional-graph.md) requires each payload type to describe how conflicts are resolved when parallel transactions are encountered.
+A DID Document is not a CRDT, but its contents are. The paragraphs below describe how each of the elements should be treated in case of a conflict.
+First we describe the mechanism of detecting and resolving conflicts. DID Documents do not refer to their previous version in any way. 
+The transactions described in [RFC004](rfc004-verifiable-transactional-graph.md) do refer to previous transactions. This system can be used to detect conflicts and to resolve them.
+For DID Documents additional transactional requirements are added:
+
+* Updates to DID Documents MUST refer in their transaction to the transaction of the previous version. They SHOULD also refer to the latest transactions as described by [RFC005](rfc005-distributed-network-using-grpc.md).
+* In case of a conflict, the conflict can be resolved with an additional transaction referring to the all the conflicting transactions.
+
+## 5.1 Conflict detection
+
+The algorithm for conflict detection can be summarized as follows:
+
+- A DID Document update transaction is received.
+- The current version is resolved together with the metadata (the metadata holds the originating transaction(s)).
```

Comment:
clarify: what are the "originating transactions"? All transactions that lead to the current version, or just its previous transactions?